app/src/contexts/observationsContext.tsx

```diff
@@ -0,0 +1,175 @@
+import { useGridApiRef } from '@mui/x-data-grid';
+import { GridApiCommunity } from '@mui/x-data-grid/internals';
+import { useBiohubApi } from 'hooks/useBioHubApi';
+import { createContext, PropsWithChildren, useContext } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { SurveyContext } from './surveyContext';
+
+export interface IObservationRecord {
+  observation_id: number | undefined;
+  speciesName: string | undefined;
+  samplingSite: string | undefined;
+  samplingMethod: string | undefined;
+  samplingPeriod: string | undefined;
+  count: number | undefined;
+  date: Date | undefined;
+  time: string | undefined;
+  latitude: number | undefined;
+  longitude: number | undefined;
+}
+
+export interface IObservationTableRow extends Partial<IObservationRecord> {
+  id: string;
+}
+
+export const fetchObservationDemoRows = async (): Promise<IObservationRecord[]> => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve([
+        {
+          observation_id: 1,
```

Comment:
Hard coded?

Reply:
Yeah these are just for testing purposes